Source/Utils/Inc/Utils/Math/Vector3.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2024 Steven Frost and Orion Lyau
+
+#pragma once
+
+// -----------------------------------------------------------------------------
+
+namespace Utils
+{
+
+// -----------------------------------------------------------------------------
+
+struct Vector3
+{
+public:
+
+	Vector3();
+	Vector3( double X, double Y, double Z );
```

Comment:
Doesn't change the behaviour at all but could mark these params as const to indicate that the constructor doesn't change them.